tests/requirements/apk-requirements.txt

```diff
@@ -2,20 +2,9 @@ bash
 curl
 git
 openssh-client
-device-mapper
 py-pip
 iptables
-gcc
-libffi-dev
-libc-dev
-openssl-dev
 make
 python3
-python3-dev
-e2fsprogs-extra
```

Comment:
I would have thought that e2fsprogs was used by mender-artifact (debugfs). But maybe it never calls the sub commands that use it.

Reply:
I also suspected they were used... but the evidence of all tests passing in here, in 3.4.x and in 3.3.x proved otherwise :)